src/planning/angular/US2/pt/US2.md

```diff
@@ -0,0 +1,9 @@
+# Lista de Filmes
+
+## Descrição
+
+Como usuário, quero visualizar o catálogo de filmes em linhas e colunas.
+
+## Definição de Concluído
+
+- [ ] Ao visitar a aplicação implantada, deve-se visualizar em uma tabela (linhas e colunas) o catálogo de filmes da [The Movie Database API V3](https://developer.themoviedb.org/docs).
```

Comment:
- [ ] Ao visitar a aplicação implantada, deve-se visualizar em uma grade (linhas e colunas) o catálogo de filmes da [The Movie Database API V3](https://developer.themoviedb.org/docs).